src/main/java/com/greenfoxacademy/goddesstribesbackend/controllers/KingdomController.java

```diff
@@ -25,7 +25,7 @@ public KingdomController(KingdomService kingdomService) {
     this.kingdomService = kingdomService;
   }
 
-  @ApiImplicitParams({@ApiImplicitParam(name = "token", value = "Authorization token",required = true, dataType = "string", paramType = "header")})
+  @ApiImplicitParams({@ApiImplicitParam(name = "token", value = "Authorization token", required = true, dataType = "string", paramType = "header")})
```

Comment:
These formatting change do not seem related to the ticket.

Reply:
Nyomtam egy CTRL + ALT + L-et, ez lett az eredménye...